### PR TITLE
Fix broken links to supported languages in README.md

### PR DESCRIPTION
**Pull request by ustun-ed**
See discussion at https://github.com/FormidableLabs/prism-react-renderer/issues/190 which was probably closed by mistake?

### CODE REVIEW

File: README.md
Comment (ustun-ed) [inline README.md:38]: this is the result of running `doctoc`

**Review by carloskelly13 — APPROVED** [review]

**Comment by ustun-ed** [comment]
thanks @carloskelly13 . I don't see the merge button after the approval for some reason, could you merge?

**Pull request merged**
